src/main/frontend/src/pages/Account.js

```diff
@@ -1,13 +1,65 @@
-import { useContext } from "react";
-import { AuthContext } from "../components/loginAndRegistration/AuthContext.js";
-import Profile from "../components/loginAndRegistration/Profile.js"
+import { useContext, useEffect, useState } from "react";
+import { useHistory, useParams } from "react-router";
+import { AuthContext } from "../utils/AuthContext.js";
+import Bids from "../components/profilePage/Bids.js";
+import Profile from "../components/profilePage/Profile.js"
+import Seller from "../components/profilePage/Seller.js";
+import Settings from "../components/profilePage/Settings.js";
+import styles from "./Account.module.css";
+import { ACCOUNT_SECTIONS } from "../constants";
+import PageLayout from "../components/PageLayout.js";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { Link } from "react-router-dom";
+import { faArrowRight } from "@fortawesome/free-solid-svg-icons";
+import { BsFillPersonFill as ProfileIcon, BsListUl as SellerIcon } from "react-icons/bs";
+import { BiDollarCircle as BidsIcon } from "react-icons/bi";
+import { IoSettingsOutline as SettingsIcon } from "react-icons/io5";
 
 function Account(){
+
     const { loggedIn } = useContext(AuthContext);
+    const { section } = useParams();
+    const [selectedSection, setSelectedSection] = useState(section.charAt(0).toUpperCase() + section.substr(1).toLowerCase());
```

Comment:
can we create util method convertToTitleCase = (string) => {
return string.charAt(0).toUpperCase() + string.substr(1).toLowerCase()
}
and use it here as convertToTitleCase(section). Same on ln 41